scripts/manifests/firefox.json

```diff
@@ -37,7 +37,8 @@
     "storage",
     "<all_urls>",
     "menus",
-    "tabs"
+    "tabs",
+    "notifications"
```

Comment:
The permissions.md file in assets needs to be adjusted to include an explanation of _why_ we need that permission.

It is fair to use it for the proposed use case though, I guess.